package.json

```diff
@@ -14,10 +14,12 @@
   },
   "dependencies": {
     "@tinymce/tinymce-vue": "^2.0.0",
+    "@types/dropzone": "^5.0.6",
```

Comment:
Move to devDependencies, since we don't want to publish this repo as a library yet.

Reply:
@types/dropzone move